.changeset/gold-kings-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/language-common': minor
```

Comment:
This is probably a patch fix because we are not introducing breaking changes (so it's not a major), nor are we adding stuff to the API (so it's not a minor)